features/promotion/applying_promotion_rules/receiving_discount_based_on_nth_order_for_guest_customer.feature

```diff
@@ -0,0 +1,36 @@
+@applying_promotion_rules
+Feature: Receiving discount based on nth order
+    In order to pay less while an placing order
+    As a Visitor
+    I want to receive a discount for my purchase
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "PHP T-Shirt" priced at "€100.00"
+        And the store ships everywhere for free
+        And the store allows paying offline
+        And there is a promotion "1st order promotion"
+        And it gives "€20.00" off customer's 1st order
+
+    @todo
+    Scenario: Receiving a discount on an first order
+        Given I have product "PHP T-Shirt" in the cart
+        When I complete shipping address with email "john.doe@example.com"
```

Comment:
This does not sound right, do we have such step in repo? Should be something like:

When I complete addressing step with e-mail "john.doe@example.com"

WDYT?